koji_containerbuild/plugins/builder_containerbuild.py

```diff
@@ -297,7 +301,9 @@ def osbs(self):
                 os_conf = Configuration(conf_section='scratch')
                 build_conf = Configuration(conf_section='scratch')
             self._osbs = OSBS(os_conf, build_conf)
-            assert self._osbs
+            if not self._osbs:
+                msg = 'Could not successfully instantiate `osbs`'
```

Comment:
Why ` here when in previous msg " were used ?

Reply:
`osbs` is an actual class here